src/Adapter/Configuration/LogsConfiguration.php

```diff
@@ -62,6 +76,17 @@ public function getConfiguration()
     public function updateConfiguration(array $configuration)
     {
         if ($this->validateConfiguration($configuration)) {
+            $checkEmails = explode(',', $configuration['logs_email_receivers']);
+            $errors = [];
+            foreach ($checkEmails as $email) {
+                if (!$this->validate->isEmail($email)) {
+                    $errors[] = $this->translator->trans('Invalid email address.', [], 'Admin.Notifications.Error') . ' ( ' . $email . ' ) ';
```

Comment:
```suggestion
                    $errors[] = $this->translator->trans('Invalid email address %email%.' , ['%email%' => $email], 'Admin.Notifications.Error');
```

Reply:
@PululuK Why not using the same wording as in https://github.com/PrestaShop/PrestaShop/pull/24622/files ?

line 79, file 92